cookbook/recipes/subscriptions/patches/CartLineItem.tsx.8e657b.patch

```diff
@@ -0,0 +1,38 @@
+index 26102b61..4ec8324b 100644
+--- a/templates/skeleton/app/components/CartLineItem.tsx
++++ b/templates/skeleton/app/components/CartLineItem.tsx
+@@ -3,8 +3,8 @@ import type {CartLayout} from '~/components/CartMain';
+ import {CartForm, Image, type OptimisticCartLine} from '@shopify/hydrogen';
+ import {useVariantUrl} from '~/lib/variants';
+ import {Link} from '@remix-run/react';
+-import {ProductPrice} from './ProductPrice';
```

Comment:
minor - but is there a way to standardize imports to always either `'./ComponentName` or `'~/components/ComponentName'` - so that the diffs will be more accurate?